src/_colors.css

```diff
@@ -0,0 +1,39 @@
+:root {
```

Comment:
The two different versions of `white-offset` are a result of us trying to differentiate between the Belly SMB brand and BLUE. Thus, `#f7f7f7` is used on bellycard.com/SMB product and `#fafafa` is used for BLUE products. Knowing that, what is your suggestion?